### ISSUE TITLE
Running mlperf inference benchmarks on systems without internet connectivity

### ISSUE THREAD

**Issue by arjunsuresh** [mlcommons/ck#710]
Some edge systems may not be connected to internet and we need a way to run mlperf inference benchmarks on them using CM.

**Comment by gfursin** [comment]
I believe many of our CM automation recipes can plug in models and data sets locally, right?

**Comment by arjunsuresh** [comment]
yes Grigori. But this issue is for running MLPerf inference on systems without internet and there even pip install will fail. So, we probably needs to do the preparation on a host machine and then do the final run on the target. Nvidia do this for their Jetson Orin submission and not having internet improves the power efficiency :)
